packages/s2-core/src/cell/table-col-cell.ts

```diff
@@ -113,9 +115,102 @@ export class TableColCell extends ColCell {
     renderDetailTypeSortIcon(
       this,
       spreadsheet,
-      x + cellWidth - iconSize - iconMargin,
+      x + cellWidth - iconSize - iconMarginRight,
       textY,
       key,
     );
   }
+
+  protected initCell() {
+    super.initCell();
+    this.addExpandColumnIconAndTipsLine();
```

Comment:
这样就够了吧，毕竟他们是一组的，有icon就有line，没必要说明一下
```suggestion
    this.addExpandColumnIcon();
```